### PR TITLE
Fix VSCodium Insiders detection on Windows

### PR DESCRIPTION
**Pull request by voidei**
## Description

- Swap `displayNamePrefix` from `string `to `string[]`
- Create function to test `string.startswith `against strings array
- Change `editor.displayNamePrefix.startswith(display)` to aforementioned new function

## Release notes

Notes: Minor changes to editor detection and fixed detection of VSCodium Insiders for Windows

### CODE REVIEW

File: app/src/lib/editors/win32.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ type WindowsExternalEditor = {
   readonly registryKeys: ReadonlyArray<RegistryKey>
 
   /** Prefix of the DisplayName registry key that belongs to this editor. */
-  readonly displayNamePrefix: string
+  readonly displayNamePrefix: string[]
```
Comment (sergiou87) [inline app/src/lib/editors/win32.ts]: This should be `displayNamePrefixes` now

**Review by sergiou87 — CHANGES_REQUESTED** [review]
Thanks for your contribution! 💖 

Would you mind explaining in the PR description what changed in VSCodium Insiders display name to require/justify these changes?

**Comment by voidei** [comment]
@sergiou87 Changes made were so that both the new and old VSCodium Insiders could be detected, as despite fixing most issues in #15585 the user install of insiders had a different display name; as for the new funtion added, I did it so `displayNamePrefixes` could remain as a prefix, rather than matching the entire string - largely due to VSCodium Insiders' display name having "Insiders" in parenthesis on the old version, and without them in the new.

**Review by sergiou87 — APPROVED** [review]
Looks good and works as expected, thank you!

**Pull request merged**